src/templates/index.html

```diff
@@ -1,90 +1,110 @@
-For example, you would like to upgrade
-<code class="scrollable">http://github.com/tensorflow/docs/blob/master/site/en/tutorials/keras/save_and_restore_models.ipynb</code>
-you would need to replace url to
-<code class="scrollable">http://tf2up.ml/tensorflow/docs/blob/master/site/en/tutorials/keras/save_and_restore_models.ipynb</code>
-<a href="http://tf2up.ml/tensorflow/docs/blob/master/site/en/tutorials/keras/save_and_restore_models.ipynb" target="_blank">Example</a></a>
-
-<h4><a href="#bookmarklet">Bookmarklet</a></h4>
-Alternatively, you can simply use a <a href="https://en.wikipedia.org/wiki/Bookmarklet">Bookmarklet</a> for this.
-
-Feel free to drag this one 👉 <a id="bookmarklet" href="javascript:(function()%7Bwindow.open(window.location.toString().replace(%2F.*github%5C.com%5C%2F(.*)%2F%2C'http%3A%2F%2Ftf2up.ml%2F%5C%241'))%7D)()">TF2up</a><br>
 
-You can manually add a bookmarklet based on the code below. Just encode the code with <code>encodeURI</code> in your browser console.
-<code class="scrollable">
-javascript:(function(){window.open(window.location.toString().replace(/.*github\.com\/(.*)/,'http://tf2up.ml/\$1'));})()
-</code>
+    <a class="logo" href="//tf2up.ml">
+        <img src="/static/images/tf2.svg">
+    </a>
 
-Now, once you visit a GitHub repository with <code>ipynb</code> notebook, you can just click on the bookmarklet.<br>
-Next, you are going to be redirected to page with a diff, showing the difference between original notebooks and a converted one.
+    <a href="https://twitter.com/share?ref_src=twsrc%5Etfw" class="twitter-share-button" 
+        data-text="http://tf2up.ml - @TensorFlow 2.0 upgrader service, even easier upgrade to #TensorFlow 2.0 by @lc0d3r" 
+        data-hashtags="TensorFlow" data-related="lc0d3r" data-show-count="false">Tweet</a>
 
-<h2><a href="#demo">Demo</a></h2>
-<img id="demo" src="https://camo.githubusercontent.com/82231f37f147fc4ee392bac01c627ed93b1aa6a0/687474703a2f2f672e7265636f726469742e636f2f706232307a38726b59302e676966" >
-
-
-<h2><a href="#how-it-works">How it works</a></h2>
-This project is standing on the shoulders of giants:
-<ul>
-    <li> TensorFlow's <a href="https://github.com/tensorflow/tensorflow/tree/master/tensorflow/tools/compatibility" target="_blank">tf_upgrade_v2</a> with brand new support of <a href="https://github.com/tensorflow/tensorflow/pull/25680">inline jupyter notebook conversion</a></li>
-    <li><a href="https://github.com/jupyter/nbdime" target="_blank">nbdime</a> for generating nice diffs for before and after</li>
-    <li><a href="https://kubernetes.io/" target="_blank">Kubernetes + Helm</a> for deploying this tiny service</li>
-</ul>
-
-Notes
-<ul>
-    <li>We cache files on the server, so if you have updated GitHub source, we can still show you old content</li>
-</ul>
-
-<h2><a href="#maker">Maker</a></h2>
-The service was developed after work and on weekends by <a href="https://twitter.com/lc0d3r">Sergii Khomenko</a><br>
-If you
-<ul>
```

Comment:
seems like the list is also got lost, but I am fixing it in a different commit, so no worries :)

Reply:
I intentionally removed it though, since there's only one item in the list. 😅